### PR TITLE
Clarify Google Workspace AlertCenter

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

UI in docs doesn't match product. @duchatran working with CON to investigate. Until then, pausing this PR.

Source of request: https://github.com/SumoLogic/sumologic-documentation/issues/1062

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md [hunk 1]
```diff
@@ -28,25 +28,31 @@ In this configuration, you'll set up a Google service account, and configure the
...
-5. JSON for the service account is automatically downloaded. To see what the JSON looks like, and how the JSON fields map to the fields you'll configure, see [Service account JSON](#example-of-service-account-json) below.
-6. Add domain-wide delegation to your service account using the generated client ID in the above step 4 of the same section.
-7. Add the OAuth scope to the service account, from the admin console, and select it in the input form. The OAuth scope for alert API is `https://www.googleapis.com/auth/apps.alerts`. To add a scope, refer to the step-by-step process [here](https://developers.google.com/admin-sdk/directory/v1/guides/delegation#delegate_domain-wide_authority_to_your_service_account).
-:::note
-If you don't add the scope to the service account you won't be authorized to fetch alert details.
-:::
+To create Google Workspace AlertCenter service account credentials:
+
+1. From Google Workspace, select your project or create a new one.
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md:33]: From Google Cloud [console](https://console.cloud.google.com/)

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md [hunk 2]
```diff
@@ -28,25 +28,31 @@ In this configuration, you'll set up a Google service account, and configure the
...
+1. Enable **Google Workspace Alert Center API** for the Alert API. To locate this setting, you can search for "Google Workspace Alert Center API" in the search bar. Then select the **Enable** button. <br/><img src={useBaseUrl('img/send-data/google_workspace_alert_center_API_sdk.png')} alt="api-sdk" width="450"/>
+1. You will be redirected to the dashboard page. Select the **Credentials** tab in the left panel. <br/><img src={useBaseUrl('img/send-data/google_workspace_credentials.PNG')} alt="credentials" width="200"/>
+1. Click **Create Credentials**, and select **Service Account** to create service account credentials. Later you'll supply the account details and click **Done** to create a service account. <br/><img src={useBaseUrl('img/send-data/google_workspace_service_account.PNG')} alt="<service-account>" width="400"/>
+1. Navigate to the **Keys** tab on the same page. To create JSON for the service account, you must create a key. Click **Add key** and select **Create new key**. At the prompt, select **JSON** and click **Create**. <br/><img src={useBaseUrl('img/send-data/google_workspace_service_account_key.PNG')} alt="service-account-key" width="<insert-pixel-number>"/>
```
Comment (duchatran) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md:37]: This looks different from my end, could be either a different type of account, or the console has changed. We will need another person to look as well (via a Jira).

Comment (stacykor) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md:37]: @duchatran should we file the JIRA against Himanshu or is there a default apps owner?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md (hunk 1 above)
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md:33]:
```suggestion
1. From [Google Cloud console](https://console.cloud.google.com), select your project or create a new one.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md (hunk 2 above)
Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md:37]: @duchatran [Here's the ticket](https://jasklabs.atlassian.net/browse/CON-1492). Let us know who to assign to.

Comment (kimsauce) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter.md:37]: Thank you Duc for assigning to Parthiv. I'm going to push out the existing changes I've made here as they're just formatting and grammar-related. Added a note to the Jira ticket about next steps after you've resolved the UI discrepancies.

**Pull request merged**